c7n/resources/awslambda.py

```diff
@@ -252,7 +252,7 @@ def process(self, resources, event=None):
 @AWSLambda.filter_registry.register('kms-key')
 class KmsFilter(KmsRelatedFilter):
 
-    RelatedIdsExpression = 'KMSKeyArn'
+    RelatedIdsExpression = 'KmsKeyArn'
```

Comment:
This is a consequence of inconsistencies between service APIs and Security Hub... the `RelatedIdsExpression` needs to stay as `KMSKeyArn` here because that's how it comes back [from the Lambda API](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/lambda.html?highlight=KMSKeyArn#Lambda.Client.get_function).